coders/xcf.c

```diff
@@ -227,7 +228,50 @@ typedef enum
   GIMP_DIVIDE_MODE,
   GIMP_DODGE_MODE,
   GIMP_BURN_MODE,
-  GIMP_HARDLIGHT_MODE
+  GIMP_HARDLIGHT_MODE,
```

Comment:
Could you change this to use the gimp naming instead: http://www.manpagez.com/html/libgimp/libgimp-2.10.6/libgimp-gimpenums.php?

Reply:
Done in 16b7ffa